.travis.yml

```diff
@@ -11,6 +11,7 @@ before_install:
   - gem --version
   - bundle -v
 script:
+  - 'bundle exec puppet --version'
```

Comment:
I needed to know exactly which version of puppet was being tested.  We already output the ruby version so puppet version was additional info I needed as well.  `PUPPET_VERSION ='~3.0'` can mean anything.